lib/client/order.rb

```diff
@@ -64,7 +92,13 @@ def get_coupons(order_id, params = {})
           response = client.get(uri({ order_id: order_id }, 'coupons'), params)
           return response.body
         rescue Faraday::Error => e
-          raise BigcommerceApiError.new('get order coupons', { order_id: order_id }, e
+          raise BigcommerceApiError.new(
+            e.response[:status],
+            'get order coupons',
+            "Failed to get order coupons #{order_id}",
+            order_id,
+            { order_id: order_id },
```

Comment:
Drop the `order_id` in the data object, and add the BC errors.

Reply:
@mrogers-5s resolved